src/containers/Root.tsx

```diff
@@ -24,6 +24,11 @@ class Root extends React.Component {
               <Route exact={true} path="/charts" component={ChartList} />
               <Route exact={true} path="/charts/:repo" component={ChartList} />
               <Route exact={true} path="/charts/:repo/:id" component={ChartView} />
+              <Route
```

Comment:
An issue with this is when first switching from `/charts/:repo/:id` to `/charts/:repo/:id/versions/:version`, the ChartView component gets remounted rather than receiving new props, so you see a page refresh. Is there a way to avoid this?